src/components/Theme/ScreenLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import { h, Fragment, ComponentType, ComponentChildren, VNode } from 'preact'
+import theme from './style.scss'
+
+type ScreenLayoutProps = {
+  children: ComponentChildren
+  actions?: ComponentType | VNode
+}
+
+const ScreenLayout = ({ children, actions }: ScreenLayoutProps) => {
+  return (
+    <Fragment>
+      <div className={theme.scrollableContent}>{children}</div>
+      {actions && actions}
```

Comment:
Ditto here, as `actions` is either `VNode` or `undefined`, we don't need to check its presence here
```suggestion
      {actions}
```